### DISCUSSION TITLE
Will Copilot change its AI model?

### DISCUSSION THREAD

**Discussion by hikionori** [community/community#123831] [Copilot Conversations]
As far as I know, Copilot works on the basis of the Codex which is based on GPT-4, but recently OpenAI showed its new model, which is better in everything and even cheaper. This leads to the question: will Copilot change its model and, if so, what opportunities will it give and, frankly speaking, As a developer, the capabilities of the new model scare me. But I would like to see the capabilities of Copilot with it.

**Comment by Ethan-Software-Developer** [comment]
Yes at some point since they are probably gonna intergrate chat gpt in it at some point
